pandas/core/ops.py

```diff
@@ -370,16 +370,44 @@ def _get_op_name(op, special):
 e  NaN  2.0
 """
 
+_sub_example_FRAME = """
+>>> a = pd.DataFrame([2, 1, 1, np.nan], index=['a', 'b', 'c', 'd'],
+                      columns=['one'])
```

Comment:
You need to add `...` at the beginning of the line here.

The same for `b` below